docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -167,6 +167,8 @@ NOTE: {endpoint-sec} has a different output matrix.
 |{n}
 |===
 
+Support for outputing to remote {es} clusters in {fleet}-managed agents is under consideration for a future release.
```

Comment:
I had to read this a few times before it made sense. I think you're saying that Fleet-managed agents must send data to the same {es} cluster where {fleet} is running.

Maybe it would be good to state that up front and then say that support for sending to remote Elasticsearch clusters is under consideration for a future release. Or you can add your statement as a [footnote](https://docs.asciidoctor.org/asciidoc/latest/macros/footnote/) so that it's clear you are referring to the caveats listed in the table. Only problem with footnotes is that they end up on the very bottom of the page (rather than after the table).

Reply:
@dedemorton I adjusted the wording here https://github.com/elastic/observability-docs/pull/943/commits/71333463664c275af3ac78de2583e6b9f1130a19 is that better?